libraries/joomla/database/driver/pgsql.php

```diff
@@ -151,6 +151,23 @@ public function getConnectionCollation()
 		return $array[0]['lc_collate'];
 	}
 
+	/**
+	 * Internal function to get the name of the default schema for the current PostgreSQL connexion.
```

Comment:
```suggestion
	 * Internal function to get the name of the default schema for the current PostgreSQL connection.
```
We use English, not French in comments ;-)